recipes/sony_custom_layers/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "sony_custom_layers" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sony/custom_layers/archive/v{{ version }}.tar.gz
+  sha256: 43a58a319484169daaeebb7a3a5079faf8a6fbffe3748968cd68754671121ffa
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: true  # [win or py<38]
```

Comment:
Does this package actually not support windows, or is it just because of the availability of tensorflow on conda-forge? If it's the latter, then this isn't necessary and you can make this `noarch: python` and pin just the python version.